Makefile

```diff
@@ -131,23 +131,26 @@ endif
 GO_TARGET_PHONY += $(1)
 endef
 
-ifeq ($(TARGET_PLATFORM),)
+BUILD_INDOCKER_ARG := --env IN_DOCKER=1 --env HTTP_PROXY=${HTTP_PROXY} --env HTTPS_PROXY=${HTTPS_PROXY} --env GOPROXY=${GOPROXY} --volume $(ROOT):/mnt --user $(shell id -u):$(shell id -g)
+
+ifneq ($(TARGET_PLATFORM),)
+	BUILD_INDOCKER_ARG += --platform=linux/$(TARGET_PLATFORM)
+	DOCKER_BUILD_ARGS += --platform=linux/$(TARGET_PLATFORM) --build-arg TARGET_PLATFORM=amd64
```

Comment:
`--build-arg TARGET_PLATFORM=amd64`, why not use `$TARGET_PLATFORM` here?

Reply:
Sorry. It's a typo :(